specs/method-identifiers.html

```diff
@@ -305,7 +305,7 @@
         <p>This section relies on several other underlying specifications.</p>
         <dl>
           <dt>URL</dt>
-          <dd>The terms <em>URL</em>, <em>absolute URL</em>, <em>URL parser</em>, and <em>URL equivalence</em> are defined by [[!url-1-20141209]] <small><em>(or the editor's draft)</em></small>.</dd>
+          <dd>The terms <a>URL</a>, <a>absolute URL</a>, <a>URL parser</a>, and <a>URL equivalence</a> are defined by [[!url-1-20141209]] <small><em>(or the editor's draft)</em></small>.</dd>
         </dl>
```

Comment:
Actually no - respec does not resolve references to definitions in other documents.  We have some plug-ins that can do that if it is something we are interested in.  Normally the way people handle this is to have a common file of terms that is included in each document dynamically.  There is a plugin for this as well.  It removes terms that are unused from each document to reduce clutter.